Yoast/wpseo-news
# Bump plugin-grunt-tasks, remove CSS

## Context


* This pull relies on [this pull](https://github.com/Yoast/wordpress-seo/pull/15550/files) being merged into Yoast SEO.

## Summary

This PR can be summarized in the following changelog entry:

* Plugin no longer loads its own CSS file.

## Relevant technical choices:

* Added one CSS class to the new admin styling classes, dropped the CSS file from this plugin.

## Test instructions

This PR can be tested by following these steps:

* Update, see the Yoast meta box -> News SEO  content looks good.
